Simulator/Core/Simulator.cpp

```diff
@@ -89,11 +89,20 @@ void Simulator::loadParameters()
       deltaT_ = DEFAULT_dt;
    }
 
+   // Note: This might be unnecessary, but to ensure that consoleLogger is initialized
+   consoleLogger_ = log4cplus::Logger::getInstance(LOG4CPLUS_TEXT("console"));
+
```

Comment:
`Simulator` already has a `consoleLogger_` member variable, which is assigned to in the constructor. So, it is safe to use.